### DISCUSSION TITLE
The Installation part of  copy these scripts into  project.

### DISCUSSION THREAD

**Discussion by Bidaya0** [batect/batect#1188] [Ideas]
Referring to the installation tutorial of docker-compose, is it better to let users put it in `/usr/local/bin/batect` ?

People just leave docker-compose.yml in the project without bringing docker-compose.

**Comment by charleskorn** [comment]
Thanks for the question @Bidaya0. One of Batect's goals is to enable "clone repo and go" scenarios, where someone can start working with a codebase without having to install a bunch of tools. 

Placing Batect's wrapper script in your project (rather than installing it globally) allows anyone to quickly start work on your codebase and ensures that everyone is using exactly the same thing.
